ChecksFile.wl

```diff
@@ -152,7 +152,8 @@ Print[Im[\[CapitalOmega]ch[2.,1.,0.,0]],Im[evtest]];
 
 
 (*Check the dispersion relation from Gail's paper*)
-dispersionCheck[data_,\[CapitalOmega]_,k_]:=Module[{\[Phi]0,\[Phi]1,\[CapitalOmega]p,kp,check,Idis,\[Theta]},
+Options[dispersionCheck]={"output"-> "check"};

```

Comment:
The better solution would be to have a separate function that outputs the Is, and have the dispersionCheck function call that one. Once again, one function should have one task.

Reply:
Sure. This is a result of my changing my mind about how I wanted to do it. At first I wanted to pass the I's to the alldispersion function and compute the I's there, but that turned out to be slow. I'll separate these.